Control/Monad/Logic.hs

```diff
@@ -23,6 +23,11 @@
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE RankNTypes            #-}
 {-# LANGUAGE UndecidableInstances  #-}
+{-# LANGUAGE DeriveTraversable     #-}
+#if __GLASGOW_HASKELL__ < 710
```

Comment:
It does not hurt to enable `DeriveFoldable` / `DeriveFunctor` unconditionally. Also please sort language extensions alphabetically.

Reply:
Sure.

Reply:
BTW: the (only) reason to enable them conditionally is that if the minimal GHC version ever rises to 7.10, it'll be easier to remove compat cruft if it's guarded by CPP. If that's not important to you, I can certainly enable those unconditionally.

Reply:
Please enable them unconditionally.